pages/pipelines/trigger_step.md.erb

```diff
@@ -9,6 +9,18 @@ steps:
   - trigger: "another-pipeline"
 ```
 
+All builds created by a trigger step will be authored by the same user that created the initial build. This user must:
+
+* be a member of your organisation
+* have a verified email address
+
+If you have [Teams](/docs/pipelines/permissions) enabled in your organisation, the authoring user must also have:
+
+* 'Build' permission on *every* pipeline that will be triggered
+
+If the author does not have the required access, the initial build will fail, and builds on subsequent pipelines will not be triggered.
```

Comment:
“initial” build feels like kind of confusing wording

Reply:
"If the author doesn't have sufficient access to the pipeline being triggered, the trigger step will fail and cause the original build to fail as well"?